eslint.config.js

```diff
@@ -0,0 +1,6 @@
+import eslintConfig from '@terascope/eslint-config';
+// need to probably put this in original eslint-config
+// don't lint the dist folder
+eslintConfig[0].ignores.push('dist/');
```

Comment:
The asset has a dist folder too.
```suggestion
eslintConfig[0].ignores.push('dist/');
eslintConfig[0].ignores.push('asset/dist/');
```

Reply:
I put that in though its not needed for this, the build just happens to keep this in line, as mentioned in the comments, I will be removing these and putting them back in the eslint-config package itself so this is temporary